pkg/defaults/defaults.go

```diff
@@ -330,45 +335,60 @@ func stepForTest(
 	inputImages inputImageSet,
 	c *api.TestStepConfiguration,
 	imageConfigs *[]*api.InputImageTagStepConfiguration,
-) ([]api.Step, error) {
+	pullSecret *coreapi.Secret,
+) ([]api.Step, bool, error) {
+	var hasReleaseStep bool
 	if test := c.MultiStageTestConfigurationLiteral; test != nil {
 		leases := leasesForTest(test)
 		if len(leases) != 0 {
 			params = api.NewDeferredParameters(params)
 		}
+		var testSteps []api.Step
 		step := steps.MultiStageTestStep(*c, config, params, podClient, jobSpec, leases)
 		if len(leases) != 0 {
 			step = steps.LeaseStep(leaseClient, leases, step, jobSpec.Namespace)
 			addProvidesForStep(step, params)
 		}
 		if c.ClusterClaim != nil {
 			step = steps.ClusterClaimStep(c.As, c.ClusterClaim, hiveClient, client, jobSpec, step)
+			if c.ClusterClaim.As != "" {
```

Comment:
Are we going to default this to something? Or do we need users to explicitly opt into `as: latest` to get their CLI injection to work?

Reply:
I didn't think of that, but I don't see any reason we shouldn't default the cluster claim to be imported as `latest` if not specified by the user. I'll update the PR to do that.